lib/perl/Genome/Model/Build.pm

```diff
@@ -1868,8 +1868,8 @@ sub generate_send_and_save_report {
     my $email_confirmation = Genome::Report::Email->send_report(
         report => $report,
         to => $to->id,
-        from => 'apipe@'.Genome::Config::domain(),
-        replyto => 'noreply@'.Genome::Config::domain(),
+        from => $ENV{GENOME_EMAIL_NOREPLY},
```

Comment:
yep, that would work well here. I've updated the 'from' to this environment variable.